### PR TITLE
[CFM-374-fix] Hide QP at 'pending' group, if user is site admin.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-374

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module [hunk 1]
```diff
@@ -272,9 +272,12 @@ function c4m_restful_quick_post_form($group_id) {
     }
   }
 
+  $user_may_create_discussion = c4m_user_is_site_admin() ?
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: @anvmn 
please replace this with a simple `if` condition.

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: also, the variable should probably relate to quickpost (so rename the variable)

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module [hunk 2]
```diff
@@ -272,9 +272,16 @@ function c4m_restful_quick_post_form($group_id) {
     }
   }
 
+  if (c4m_user_is_site_admin()) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: @anvmn 
please add an elaborated comment.

File: project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module (hunk 1 above)
Comment (anvmn) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: This variable indicates if user has a permission to create discussion.
Comment that comes next explains how it's used with relation to QP form:

```
  // If all resources are turned off, or user is not allowed to create a
  // discussion, disable the QP form.
```

Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module]: not sure which resources you're referring to, but, by elaborating I meant to explain __when__ the user is not allowed.
The fact that the user is not allowed is trivial from the code.

**Pull request merged**